lms/djangoapps/grades/signals/signals.py

```diff
@@ -106,3 +106,14 @@
                            # score that was created.
     ]
 )
+
+
+# This Signal indicates that the user has received a passing grade in the course for the first time.
+# Any subsequent grade changes that may vary the passing/failing status will not re-trigger this event.
+# Emits course completion event
+COURSE_GRADE_PASSED_FIRST_TIME = Signal(
```

Comment:
[request to follow-through on renaming] Can you follow through on updating the name of the `course_completed` events elsewhere in this PR?

Essentially, within the core grades layer, let's be precise of what the event actually is.  Then, in a higher layer (for now, it could be at the xapi-caliper transformation layer), we can decide that a `course_grade_passed_first_time` event translates to a `course_completion` event.